spk/rhash/Makefile

```diff
@@ -0,0 +1,21 @@
+SPK_NAME = rhash
+SPK_VERS = 1.3.5
+SPK_REV = 1
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = Fredrike
+DESCRIPTION = "RHash is a console utility for calculation  and verification of magnet linksand a wide range of hash sums like  CRC32,  MD4,	 MD5,  SHA1, SHA256, SHA512,SHA3,   AICH,  ED2K,  Tiger,  DC++ TTH,  B	itTorrent BTIH,   GOST R 34.11-94,RIPEMD-160, HAS-160, EDON-R, Whirlpo	ol and Snefru." 
```

Comment:
Maybe double-check this text? I think it should be:
_RHash is a console utility for calculation and verification of magnet links and a wide range of hash sums like CRC32,  MD4, MD5, SHA1, SHA256, SHA512, SHA3, AICH,  ED2K, Tiger, DC++ TTH, BitTorrent BTIH, GOST R 34.11-94, RIPEMD-160, HAS-160, EDON-R, Whirlpool and Snefru._

Reply:
Thanks for spotting this.